packit_service/worker/jobs.py

```diff
@@ -179,8 +181,13 @@ def process_jobs(self, event: Event) -> Dict[str, HandlerResults]:
             # failed because of missing whitelist approval
             whitelist = Whitelist()
             github_login = getattr(event, "github_login", None)
```

Comment:
Can we rename the `github_login` to `user_login` as well to have the same API?

Reply:
fixed